jenkins/Jenkinsfile.nightly

```diff
@@ -62,9 +62,9 @@ pipeline {
                     docker.image(IMAGE_NAME).inside("--runtime=nvidia -v ${HOME}/.m2:${HOME}/.m2:rw \
```

Comment:
Should we remove the mounting of the typical `.m2` here?  I would think it should no longer be needed.

Reply:
Yes, need to remove it.
Done